train.py

```diff
@@ -77,13 +80,10 @@
   # Other flags.
   flags.DEFINE_integer("num_readers", 8,
                        "How many threads to use for reading input files.")
-  flags.DEFINE_string("master", "", "TensorFlow master to use.")
-  flags.DEFINE_integer("task", 0, "Task id of the replica running the training."
-                       " 0 implies chief Supervisor.""")
-  flags.DEFINE_integer("ps_tasks", 0, """Number of tasks in the ps job.
-                       If 0 no ps job is used.""")
   flags.DEFINE_string("optimizer", "AdamOptimizer",
                       "What optimizer class to use.")
+  flags.DEFINE_bool("log_device_placement", False,
+                    "Whether device placement should be logged.")
```

Comment:
How about "Whether to write the device every op will run on into the logs on startup".